src/testdrive/Cargo.toml

```diff
@@ -8,7 +8,7 @@ publish = false
 [dependencies]
 async-trait = "0.1.36"
 atty = "0.2"
-avro = { path = "../avro" }
+avro = { path = "../avro", package = "mz-avro" }
```

Comment:
Do you object to changing all the imports to `mz_avro` at some point? (This is totally fine for now, just asking if you had a moral objection or whether you were just doing the easy thing.)